README.md

```diff
@@ -135,6 +135,16 @@ The above script will build CWS, verify your configuration, then will start the
 
 You can stop CWS by running `./stop_dev.sh`. The script will bring down the console and all local workers.
 
+### Running Unit and Integration Test: cws-test
+
+Start `test.sh` script by running:
+
+```
+bash -c './test.sh'
```

Comment:
Don't need bash -c.  That is only for Jenkins.

Reply:
Fixed